web/custom-plugins/forecast-of-response/src/components/Plots/LinesOverTime.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  ReferenceLine,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts'
+import { plotColors } from '../Design/Colors'
+
+export type TLineChartDataPoint = {
+  name: string
+  [key: string]: number
+}
+
+export type TLineChartData = {
+  data: TLineChartDataPoint[]
+}
+
+export default (props: {
+  data: TLineChartData
+  graphNames: string[]
+  orangeLine: number
+  redLine: number
+}): JSX.Element => {
+  const { data, graphNames, orangeLine, redLine } = props
+
+  return (
+    <div
+      style={{ width: '100%', height: '300px', border: 'darkgrey 1px solid' }}
+    >
+      <ResponsiveContainer width="100%" height="100%">
+        <LineChart data={data}>
+          <CartesianGrid strokeDasharray="2 2" />
+          <XAxis dataKey="timestamp" />
+          <YAxis unit="m" />
+          <Tooltip />
+          <Legend />
+          {/*TODO: Read threshold values from result file*/}
+          <ReferenceLine y={redLine} stroke="red" label="Max" />
+          <ReferenceLine y={orangeLine} stroke="orange" label="Warning" />
```

Comment:
maybe rename redLine -> maxLine and orangeLine -> warningLine or something?

Reply:
fixed!